test/runAllTests.cmd

```diff
@@ -0,0 +1,8 @@
+%DNX_HOME%\runtimes\dnx-clr-win-x86.1.0.0-beta4\bin\dnx.exe .\Microsoft.ApplicationInsights.AspNet.Tests test
```

Comment:
Maybe we can just give a reference to readme of this: https://github.com/aspnet/Home

Ideally something like IF "%DNX_HOME%"=="" ECHO DNX_HOME is NOT defined. Please download the latest dnx version using instructions from https://github.com/aspnet/Home

With Powershell we can not only check, but also download required dnx version. Oleg, why do you need powershell? Is it for automation?